Jenkinsfile

```diff
@@ -1,19 +1,26 @@
 pipeline {
     agent any 
     stages {
-        stage('Settin up env') {
+        stage('Checking env') {
             steps {
-                echo "Start virtual environment"   
-                sh """ 
+                echo "Check the environment"
```

Comment:
I'd still like some labelling on the lines below. Just so as a user you can tell what each path is from the logfile.

Reply:
I misunderstood what you meant. Reverted the previous change (separate check stage) and added more comments